packages/driver/src/codecs/json.ts

```diff
@@ -74,7 +84,11 @@ export class PgTextJSONCodec extends JSONCodec {
 export class JSONStringCodec extends ScalarCodec implements ICodec {
```

Comment:
With the old customCodecs removed we can also delete this.

Reply:
good